tensorflow-core/pom.xml

```diff
@@ -84,588 +85,13 @@
       -->
       <id>javacpp-platform-host</id>
       <activation>
-        <activeByDefault>true</activeByDefault>
         <property>
-          <name>javacpp.platform.host</name>
+          <name>!javacpp.platform</name>
```

Comment:
There is no need of having this profile if we set the default value of `javacpp.platform` to `${os.name}-${os.arch}` by default in the pom `properties`, and simply let user override it if needed